flake.nix

```diff
@@ -37,6 +37,11 @@
     arion.follows = "blank";
   };
   outputs = inputs: let
+    # this is a standard-specific contract to wrap the data block type with metadata
+    dataWith = meta: data: {
+      __std_data_wrapper = true;
+      inherit data meta;
+    };
```

Comment:
I don't think this is actually made available in outputs? Looks like it only exists as a stray variable in the `let` block.

Recirculating from #255 where I ran into this error:

```console
$ std check
error: attribute 'dataWith' missing

       at /nix/store/965qmbcrgf1i0kj608gk18sf118zk2hg-cells/users/metadata.nix:10:18:

            9| in
           10|   l.mapAttrs (n: inputs.std.dataWith {description = "Metadata for user '${n}'";}) {
             |                  ^
           11|     bella = import ./bella/metadata.nix;
(use '--show-trace' to show detailed location information)
```

In `flake.nix`:

```nix
# "github:divnix/std?ref=feat-add-data-contract" doesn't work either
inputs.std.url = "github:divnix/std/feat-add-data-contract";
```

Reply:
sry, fixed in [force push](https://github.com/divnix/std/compare/a57cd3512516671b6153147511ad94b26d8563ba..0af8c0ea29ad79ffdc04277d179ba59cd639cbd4)